docs/Workshop_Areas/Lab/lbl05001.md

```diff
@@ -0,0 +1,112 @@
+# LBL05001 - Containment Level One For BioLab Users
+
+  ------
+  Approved by:             SOP No. LBL01001
+
+  Signed: 
+  \_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_    
+  Effective from:
+\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+  Date: 
+  \_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_   
+  Last edited:
+ \_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_\_
+  
+
+## Purpose
+
+In order to satisfy the [Genetically Modified Organisms (Contained Use)
+Regulations 2014](http://www.hse.gov.uk/pubns/books/l29.htm)[^1], any
+CL1 project being undertaken in the BioLab should follow the procedures
+outlined in this document in conjunction with any procedures specified
+in the risk assessment for that project .
+
+## Scope
+
+This relates to all CL1 projects.
+
+## Responsibilities
+
+This applies to anyone carrying out these procedures as part of a CL1
+project. The user should ensure that others do not inadvertently
+compromise the containment of GMOs.
+
+## Related documents
+
+ -   [Manchester Hackspace BioLab BioLab Usage
+     Guidelines](biolab-usage-guidelines.md)
+ -   [LBL04006 - Using the Autoclave for Waste
+     Deactivation](lbl04006.md)
+ -   [LBL06001 - Project Risk Assessment Procedure for Lab Users
+     Proposing New Projects](lbl06001.md)
+ -   [LBL06002 - Risk Assessment Procedure for Lab Users Proposing
+     Transport of GMOs Outside the Lab](lbl06002.md)
+
+## Definitions
+
+GMO:   Genetically Modified Organism
+
+GMM:   Genetically Modified Micro-organism
+
+CL1:   Containment Level 1
+
+## Procedures
+
+### Risk Assessment
+
+A written risk assessment must be approved before work can commence on
+any CL1 project (see [BioLab Usage
+Guidelines](biolab-usage-guidelines.md) for guidance on determining
+what is classed as CL1). The risk assessment procedure is detailed in
+[LBL06001](lbl06001.md) .
+
+### Worksite
+
+All work with GMOs as part of a CL1 project must take place within the
+BioLab. The BioLab is equipped with door access control in order to
+enable restriction of access to only trained users when appropriate. An
+observation window is also present.
```

Comment:
> I am informed the door is opaque

This would be a suitable addition, where allowable, WoodyDusty could route the door and add a perspex sealed window?